feat(ci): Build multi-arch Docker images (`amd64`, `arm64`) (#496)
antonbabenko/pre-commit-terraform: README.md

**Build from scratch**:

> > **Note**: To build image you need have [`docker buildx`](https://docs.docker.com/build/install-buildx/) enabled as default builder.  

Reviewer: ```suggestion
> **Note**: To build image you need to have [`docker buildx`](https://docs.docker.com/build/install-buildx/) enabled as default builder.  
```

> > Otherwise - provide `TARGETOS` and `TARGETARCH` as additional `--build-arg`'s to `docker build`.

Reviewer: Still don't get why don't we just set default value in Dockerfile =) This shouldn't prevent `buildx` from overriding them. Or am I getting this incorrectly?

Reviewer 2: `buildx` is required for multi-arch builds. I'd like to force users to use `buildx` when they build images from scratch, and because it's not our "core functional" there is no reason to maintain compatibility with legacy `docker build`

Reviewer 2: Buildx exist at least for 3 years

Reviewer: > `buildx` is required for multi-arch builds. I'd like to force users to use `buildx` when they build images from scratch, and because it's not our "core functional" there is no reason to maintain compatibility with legacy `docker build`

Makes sens.

> When hooks-related `--build-arg`'s is not specified, only the latest version of `pre-commit` and `terraform` will be installed.

Reviewer: ```suggestion
When hooks-related `--build-arg`s are not specified, only the latest version of `pre-commit` and `terraform` will be installed.
```
